spec/javascripts/inventories/InventoryAccessRequestSpec.js

```diff
@@ -0,0 +1,30 @@
+describe('Resource: InventoryAccessRequest', function() {
+  var subject, scope, $httpBackend;
+
+  beforeEach(function() {
+    module('PDRClient');
+    inject(function($resource, $rootScope, InventoryAccessRequest, $injector) {
+      scope = $rootScope.$new();
+      subject = InventoryAccessRequest;
+      $httpBackend = $injector.get('$httpBackend');
+    });
+  });
+
+  it("updates access request", function(){
+    $httpBackend.expectPATCH('/v1/inventories/10/access_requests', { status: 'denied' }).respond(200, {});
+    var users = subject.update({
```

Comment:
You don't do anything with this variable.  If this is meant to be used as a verification bit and we're relying on `$httpBackend` to help with that, we probably don't need the variable.